dist/init/linux-systemd/README.md

```diff
@@ -1,84 +1,127 @@
-# systemd unit for caddy
+# systemd Service Unit for Caddy
 
 Please do not hesitate to ask on
 [caddyserver/support](https://gitter.im/caddyserver/support)
-if you have any questions.
-Feel free to prepend to your question the username of whoever touched the file most recently,
-for example `@wmark re systemd: …`.
+if you have any questions. Feel free to prepend to your question
+the username of whoever touched the file most recently, for example
+`@wmark re systemd: …`.
 
-The provided file is written for **systemd version 229** or later!
+The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The easiest way to check your systemd version is to look at the version of the installed package
+(e.g. 'sudo yum info systemd' on RedHat/Fedora systems).
 
-## Quickstart
+## Instructions
 
-In the following sections, we will assume that you want to run caddy
-as user `www-data` and group `www-data`, with UID and GID 33.
-Adjust this to your liking according to the preferences of your Linux distribution!
+We will assume the following:
+
+* that you want to run caddy as user `www-data` and group `www-data`, with UID and GID 33
+* you are working from a non-root user account that can use 'sudo' to execute commands as root
+
+Adjust as necessary or according to your preferences.
+
+First, put the caddy binary in the system wide binary directory and give it
+appropriate ownership and permissions:
 
 ```bash
-groupadd -g 33 www-data
-useradd \
+sudo cp /path/to/caddy /usr/local/bin
+sudo chown root:root /usr/local/bin/caddy
+sudo chmod 755 /usr/local/bin/caddy
+```
+
+Give the caddy binary the ability to bind to privileged ports (e.g. 80, 443) as a non-root user:
+
+```bash
+sudo setcap 'cap_net_bind_service=+ep' /usr/local/bin/caddy
```

Comment:
This is dangerous, and I have written the unit files for systemd specifically in a way that does no require this.

Again, do not use `setcap`. Given the prevalent cargo cult this will end on the internet and cause much grief in the future.